2-mandatory/2-financial-times.js

```diff
@@ -23,19 +40,36 @@ function titleWithFewestWords(allArticleTitles) {
     (Hint: remember that you can also loop through the characters of a string if you need to)
 */
 function headlinesWithNumbers(allArticleTitles) {
-    // TODO
+    const regex = /[0-9]/;
+    let result = [];
+    for (let articleTitle of allArticleTitles) {
+        for (let i = 0; i < articleTitle.length; i++) {
+            if(articleTitle[i].match(regex)){
+                result.push(articleTitle);
+                break;
+            } 
+        }
+    }
+    return result;
 }
+    
+
 
 /*
     The Financial Times wants to understand what the average number of characters in an article title is.
     Implement the function below to return this number - rounded to the nearest integer.
 */
 function averageNumberOfCharacters(allArticleTitles) {
```

Comment:
It looks like this implementation will work.
Again, I'd encourage you to think about variable naming - what is `averageCharacter` really storing here? Maybe `totalCharacters` might be a better name?
Also, is there an easier way to get the number of articles without using a `numberOfArticles` variable?